server/src/test/java/org/elasticsearch/index/engine/NoOpEngineTests.java

```diff
@@ -114,19 +114,21 @@ public void testNoOpEngineStats() throws Exception {
                     if (rarely()) {
                         engine.flush();
                     }
-                    globalCheckpoint.set(engine.getLocalCheckpoint());
+                    engine.syncTranslog(); // advance local checkpoint
```

Comment:
Is this necessary or a left-over?

Reply:
For the global checkpoint I like to use the persisted local checkpoint to make it more realistic.